css/civicrm.css

```diff
@@ -4594,10 +4594,6 @@ span.crm-status-icon {
   text-decoration: line-through;
 }
 
-.crm-printButton {
-  float: right;
-}
-
```

Comment:
@parag I made this change to avoid introducing new single-purpose style rules that duplicate existing generic style rules.